src/boot/ksdk1.1.0/devBME680.c

```diff
@@ -215,7 +215,7 @@ printSensorDataBME680(bool hexModeFlag, uint16_t menuI2cPullupValue)
 	unsignedRawAdcValue =
 			((readSensorRegisterValueMSB & 0xFF)  << 12) |
 			((readSensorRegisterValueLSB & 0xFF)  << 4)  |
-			((readSensorRegisterValueXLSB & 0xF0) >> 4);
+			((readSensorRegisterValueXLSB & 0x0F) >> 4);
```

Comment:
Can you explain why you think it is meaningful to mask off the the upper bits keeping only the lower four bits by `(readSensorRegisterValueXLSB & 0x0F)` and then shifting right by four to throw away the bits you just purposefully kept? Double-check your reasoning (see **Section 5.3.4.2 Temp data** of the BME680 manual).